src/main/java/no/sikt/nva/validators/ExcelScraperValidator.java

```diff
@@ -0,0 +1,41 @@
+package no.sikt.nva.validators;
+
+import static no.sikt.nva.scrapers.ExcelScraper.FIRST_ROW;
+import static no.sikt.nva.scrapers.ExcelScraper.SECOND_ROW;
+import static no.sikt.nva.validators.ExcelHeaderValidator.validateHeaders;
+import static no.sikt.nva.validators.ExcelRowValidator.validateRow;
+import no.sikt.nva.exceptions.ExcelException;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+
+public class ExcelScraperValidator {
+    public static final String ERROR_MESSAGE_EMPTY_SHEET = "Empty sheet";
+
+    public static void validate(Workbook workbook) throws ExcelException {
```

Comment:
Again: make this return a list of a object containing each row of the excel sheet.